coffee-client/src/components/offer-detail.js

```diff
@@ -0,0 +1,145 @@
+import React, { Component } from 'react';
+import styled from 'react-emotion';
+import MapDetail from './map-detail';
+const plantationPic = require('../assets/images/plantations/costa_rica.jpg')
+
+const Div = styled('div')`
+  font-family: avenir next; 
+    color: black;
+`;
+
+  const Location = styled('div')`
+  position: absolute;
+  font-weight: 300;
+  font-size: 1.3em;
+  letter-spacing: -2px;
+  top: 51%;
+  left: 5%;
+`;
+
+const Company = styled('div')`
+  position: absolute; 
+  font-weight: 600;
+  font-size: 1.7em;
+  letter-spacing: -2px;
+  top: 12%;
+  left: 5%;
+`;
+
+const ThirdTitle = styled('div')`
+  position: absolute;
+  font-weight: 220;
+  font-size: 1.1em;
+  letter-spacing: -1px;
+  right: 7%;
+  top: 7%;
+`;
+
+const Subtitle = styled('div')`
+  position: absolute;
+  font-weight: 400;
+  font-size: 1.3em;
+  letter-spacing: -2px;
+  top: 7%;
+  left: 5%;
+`;
+
+const Paragraph = styled('p')`
+  font-weight: 400;
+  font-size: 1em;
+  line-height: 1.2;
+  margin-left: 5%;
+  margin-right: 5%;
+  margin-top: ${props => props.top};
+`;
+
+const CoffeeBox = styled('div')`
+  display: flex;  
+  flex-direction: column; 
+  overflow-x: scroll;
+  height: 30%;
+  width: 52%; 
+  position: absolute;
+  letter-spacing: -0.5px;
+  top: 17%;
+  left: 5%;
+  background-color: white;
+`;
+
+const ReviewBox = styled('div')`
+  display: flex; 
+  flex-direction: column; 
+  overflow-x: scroll;
+  height: 30%;
+  width: 50%; 
+  position: absolute;
+  letter-spacing: -0.5px;
+  bottom: 11%;
+  right: 5%;
+  background-color: white;
+`;
+
+const ImageFormat = styled('div')`
+  position: absolute;
+  top: 16%;
+  left: 60%;
+  width: 38%; 
+`;
+
+
+class OfferDetail extends Component {
+
+
+
+  render() {
+    return (
+      <Div>
+    
+        <ImageFormat>   <img src={plantationPic} alt= "plantation" /></ImageFormat>
+        <Company> JUAN VALDEZ </Company>
+
+        <CoffeeBox> 
+
+          <Subtitle> Coffea Canephora</Subtitle> 
+          <ThirdTitle> PRICE - $ 200 </ThirdTitle>
+
+          <Paragraph top = '7%'> "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, 
```

Comment:
When working with mock data, I prefer you to put all this content in a structure (object, array…) and then working with it in the render method. This way all is more similar to the way it's going to work in production.

Reply:
ohh yes I see, thank you